UnitySDK/Assets/ML-Agents/Examples/PushBlock/Scripts/PushAgentBasic.cs

```diff
@@ -84,8 +84,8 @@ public override void CollectObservations()
         {
             var rayDistance = 12f;
 
-            AddVectorObs(m_RayPer.Perceive(rayDistance, m_RayAngles, m_DetectableObjects, 0f, 0f));
-            AddVectorObs(m_RayPer.Perceive(rayDistance, m_RayAngles, m_DetectableObjects, 1.5f, 0f));
```

Comment:
This doesn't look right - the args default to 0.0 but the second line is a behavior change

Reply:
fat fingered

Reply:
fixed